src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3913,6 +3913,18 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="ShowPluginsOverview_NonGlobal.Content" xml:space="preserve">
     <value>Not included in global results</value>
   </data>
+  <data name="Flyout_ShowUpdateDetails_ToolTip.Text" xml:space="preserve">
+    <value>Show update</value>
+  </data>
+  <data name="GeneralPage_ToggleSwitch_ShowNewUpdatesToast.Description" xml:space="preserve">
+    <value>If disabled the notifications in the settings and the tray flyout are still shown</value>
+  </data>
+  <data name="GeneralPage_ToggleSwitch_ShowNewUpdatesToast.Header" xml:space="preserve">
+    <value>Show Action Center notifications for new updates</value>
```

Comment:
@Jay-o-Way 
What about "Show [desktop|system] notifications for new updates"? (Then we get rid of the Notification Center vs Action Center problem.)